app/models/concerns/issue_type_validatable.rb

```diff
@@ -14,4 +15,10 @@ def location_type_should_belong_to_company
                  I18n.t("validations.common.belonging_to_company"))
     end
   end
+
+  def base_location_type_consistency
+    if base_issue_type && location_type && base_issue_type&.base_location_type != location_type&.base_location_type
```

Comment:
base_issue_type && location_type me semble superflux `base_issue_type&.base_location_type != location_type&.base_location_type` suffit à verifier ce que tu veux verifier , aussi fauddrait renommer la méthode de sorte que ça soit claire et concis à l'instar de location_type_should_belong_to_company

Reply:
c'est noté, je renommerai la méthode au prochain commit